src/cljs/athens/hotkeys.cljs

```diff
@@ -0,0 +1,75 @@
+(ns athens.hotkeys
+  (:require
+    ["react-hotkeys" :refer [GlobalHotKeys HotKeys configure]]
+    [re-frame.core :refer [dispatch subscribe]]
+    [athens.util :as util]
+    [athens.router :as router]))
+
+(configure #js{:ignoreTags #js[]})
```

Comment:
Instead of automatically configuring during ns load, maybe expose it as dedicated fn like `init` or `config`.